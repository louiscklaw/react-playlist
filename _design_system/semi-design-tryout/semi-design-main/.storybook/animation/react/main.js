const config = require('../../base/base');

module.exports = {
  ...config,
  "stories": [
    '../../../packages/(semi-animation-react|semi-animation-styled)/_story/*.react.stories.(js|jsx)',
  ],
  typescript: {
    check: false,
    checkOptions: {}
  },
};
