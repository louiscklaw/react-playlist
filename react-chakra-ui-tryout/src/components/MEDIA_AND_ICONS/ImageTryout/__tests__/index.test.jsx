import React from 'react';
import { render } from '@testing-library/react';
import renderer from 'react-test-renderer';

import { ImageTryout } from '../';

describe('test badge tryout', () => {
  test('helloworld', () => {
    const tree = renderer.create(<ImageTryout />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
