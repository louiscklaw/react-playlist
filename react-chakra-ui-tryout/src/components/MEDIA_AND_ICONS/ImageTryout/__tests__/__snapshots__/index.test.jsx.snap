// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test ImageTryout tryout helloworld 1`] = `
<div>
  ImageTryout
  <div>
    Usage 
    <div
      className="css-1r1v3mz"
    >
      <img
        alt="Segun Adebayo"
        className="chakra-image__placeholder css-0"
      />
    </div>
  </div>
  <div>
    Size 
    <div
      className="chakra-stack css-x9juev"
    >
      <img
        alt="Segun Adebayo"
        className="chakra-image__placeholder css-b471h5"
      />
      <img
        alt="Dan Abramov"
        className="chakra-image__placeholder css-82vs3h"
      />
      <img
        alt="Dan Abramov"
        className="chakra-image__placeholder css-cl5oj8"
      />
    </div>
  </div>
  <div>
    ImageWithBorderRadius 
    <img
      alt="Segun Adebayo"
      className="chakra-image__placeholder css-1oidk5m"
    />
  </div>
  <div>
    FallbackSupport 
    <img
      className="chakra-image__placeholder css-0"
      src="https://via.placeholder.com/150"
    />
  </div>
  <div>
    TestHelloworld05 
    TestHelloworld05
  </div>
  <div>
    TestHelloworld06 
    TestHelloworld06
  </div>
  <div>
    TestHelloworld07 
    TestHelloworld07
  </div>
  <div>
    TestHelloworld08 
    TestHelloworld08
  </div>
  <div>
    TestHelloworld09 
    TestHelloworld09
  </div>
  <div>
    TestHelloworld10 
    TestHelloworld10
  </div>
  <div>
    TestHelloworld11 
    TestHelloworld11
  </div>
  <div>
    TestHelloworld12 
    TestHelloworld12
  </div>
  <div>
    TestHelloworld13 
    TestHelloworld13
  </div>
  <div>
    TestHelloworld14 
    TestHelloworld14
  </div>
  <div>
    TestHelloworld15 
    TestHelloworld15
  </div>
  <div>
    TestHelloworld16 
    TestHelloworld16
  </div>
  <div>
    TestHelloworld17 
    TestHelloworld17
  </div>
  <div>
    TestHelloworld18 
    TestHelloworld18
  </div>
  <div>
    TestHelloworld19 
    TestHelloworld19
  </div>
  <div>
    TestHelloworld20 
    TestHelloworld20
  </div>
</div>
`;
